app/src/org/commcare/android/database/global/GlobalDatabaseUpgrader.java

```diff
@@ -50,6 +54,15 @@ private boolean upgradeOneTwo(SQLiteDatabase db, int oldVersion, int newVersion)
     }
 
     private boolean upgradeTwoThree(SQLiteDatabase db) {
```

Comment:
Hey, 

In my mind it only depends on whether you could do them independently.

I think if you could do the upgrade on a v3 (IE: Not-released, but the v3 that was pushed), it seems like it isn't really any more work to move the upgrade to v4 and I'm fine with having multiple-bumps in a cycle.

If you are depending on the pre-v3 state to do the operations, though, it makes sense to keep it here.